_docs/_latest/setup/kubernetes.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Deploy Forseti Security on Kubernetes
+order: 004
+---
+
+# {{ page.title }}
+
+This guide explains how to setup Forseti on Kubernetes.
+
+Note: The current work here is considered not yet ready for production use.
```

Comment:
Rather than say its not ready for production, perhaps ref to hardening approaches?

"If you decide to use this in production, consider implementing the k8s hardening recommendations,
ref. https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster

as well as CIS Benchmarks for GCP, section 7, Kubernetes
ref. https://learn.cisecurity.org/benchmarks"

Reply:
Great suggestion, done.